package.json

```diff
@@ -40,7 +40,7 @@
     "react-tap-event-plugin": "^2.0.0"
   },
   "peerDependencies": {
-    "material-ui": "~0.16.7",
+    "material-ui": ">=0.16 <=0.17",
```

Comment:
Why not do the following? `^0.16`

I don't see a reason to limit the peer dependency to a fixed range, unless someone reports an issue.